src/components/views/rooms/EntityTile.js

```diff
@@ -183,17 +197,18 @@ const EntityTile = React.createClass({
 
         const av = this.props.avatarJsx || <BaseAvatar name={this.props.name} width={36} height={36} />;
 
+        // The wrapping div is required to make the magic mouse listener work, for some reason.
         return (
-            <AccessibleButton className={mainClassName} title={this.props.title}
-                    onClick={this.props.onClick} onMouseEnter={this.mouseEnter}
-                    onMouseLeave={this.mouseLeave}>
-                <div className="mx_EntityTile_avatar">
-                    { av }
-                    { power }
-                </div>
-                { nameEl }
-                { inviteButton }
-            </AccessibleButton>
+            <div ref={(c) => this.container = c} >
```

Comment:
Consider having a bound method here instead to not create a function on every render.

Reply:
Not applicable anymore: using CSS